features/support/env.rb

```diff
@@ -92,7 +92,7 @@ def feature
 # Remove/comment out the lines below if your app doesn't have a database.
 # For some databases (like MongoDB and CouchDB) you may need to use :truncation instead.
 begin
-  DatabaseCleaner.strategy = :transaction
+  DatabaseCleaner.strategy = :truncation, { except: %w[document_categories] }
```

Comment:
Why are we excluding `document_categories`?

Reply:
The switch to truncation, essentially, clears the database after each feature file.
Because most of the features set up the required values in the DB, i.e. create(:provider), etc, but document categories are only created in the db:seed call that is run once.
As document_categories is a lookup, there is no danger of new records being created, or duplicates being added. 
The other option is manually adding the document categories in every scenario that needs them